### PR TITLE
Added Go to the party 🎉

### PR DESCRIPTION
**Pull request by MistaTwist**
Includes:

- package name randomizer
- random number of package imports, all real and from the standard library!
- put all generation code into the go.js file, for a nice tidy one-liner in the RandomCodeGenerator switch statement.

Apologies in advance for not calling anything from the imported packages!

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
Hello! Please could you rebase with main and fix up some conflicts? I just merged in a PR that reorganised some of the constants and things, which has made contributing easier in the long run. Thanks for your PR!

**Comment by MistaTwist** [comment]
Sure thing - I'll try and sort it in my lunch break - need to refamiliarize with rebase first :)

File: tools/RandomCodeGenerator.js [hunk 1]
```diff
@@ -73,16 +74,58 @@ export function generateRandomCode(language, lines) {
...
       }
-
+      
       lastLine = "\r\n    ()";
-
+      
       return firstLine + fillerLines.join("\n\r") + lastLine;
+    case "go":
+      // get a random amount of package imports.
+      const importstoGet = Math.floor(lines/5);
```
Comment (whitep4nth3r) [inline tools/RandomCodeGenerator.js:89]: Change to importsToGet

Comment (MistaTwist) [reply tools/RandomCodeGenerator.js:89]: This is done!

**Comment by whitep4nth3r** [comment]
@all-contributors please add @MistaTwist for code

**Pull request merged**